examples/Viewer.tsx

```diff
@@ -311,6 +312,17 @@ class Viewer extends React.Component<{}, ViewerState> {
         });
     }
 
+    public convertFile(obj: Record<string, any>, fileType: string) {
+        simulariumController.configureNetwork(netConnectionSettings);
```

Comment:
I can't remember what configureNetwork does.  is it something that might have async failure states such that you want to do 
controller.configureNetwork().then(controller.convertTrajectory())...

Reply:
Nope, `configureNetwork()` just returns void, and throws an error if it runs into one. `configureNetwork()` is also called elsewhere in this file and it looks like what I have (with no `.then()`)